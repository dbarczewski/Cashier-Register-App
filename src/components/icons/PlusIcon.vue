<template>
  <svg width="24" height="24" viewbox="0 0 24 24">
    <path
      pid="0"
      d="M22.5 10.5h-9v-9C13.5.7 12.8 0 12 0s-1.5.7-1.5 1.5v9h-9c-.8 0-1.5.7-1.5 1.5s.7 1.5 1.5 1.5h9v9c0 .8.7 1.5 1.5 1.5s1.5-.7 1.5-1.5v-9h9c.8 0 1.5-.7 1.5-1.5s-.7-1.5-1.5-1.5z"
      fill="#FFF"
      fill-rule="nonzero"
    />
  </svg>
</template>

<script lang="ts">
import { defineComponent } from "vue";

export default defineComponent({
  name: "PlusIcon",
});
</script>
