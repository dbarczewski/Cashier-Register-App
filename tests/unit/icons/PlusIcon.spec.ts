import { shallowMount } from "@vue/test-utils";
import PlusIcon from "@/components/icons/PlusIcon.vue";

describe("icons/AvatarIcon", () => {
  it("renders component correctly", () => {
    const wrapper = shallowMount(PlusIcon);
    expect(wrapper.element).toMatchSnapshot();
  });
});
