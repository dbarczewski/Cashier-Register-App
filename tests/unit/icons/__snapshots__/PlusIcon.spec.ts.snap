// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icons/AvatarIcon renders component correctly 1`] = `
<svg
  class="h-6 w-6"
  fill="none"
  stroke="currentColor"
  viewBox="0 0 24 24"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M12 6v6m0 0v6m0-6h6m-6 0H6"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="2"
  />
</svg>
`;
